<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVSGMParametersControlWidget</class>
 <widget class="QWidget" name="OpenCVSGMParametersControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCVSGMParameters</string>
  </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_0">
       <property name="text">
         <string>Pre Filter Cap</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="preFilterCapSpinBox">
       <property name="toolTip">
         <string>Pre Filter Cap</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_1">
       <property name="text">
         <string>SAD Window Size</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="sADWindowSizeSpinBox">
       <property name="toolTip">
         <string>SAD Window Size</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
       <property name="text">
         <string>P1 multiplier</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="p1MultiplierSpinBox">
       <property name="toolTip">
         <string>P1 multiplier</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
       <property name="text">
         <string>P2 multiplier</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="p2MultiplierSpinBox">
       <property name="toolTip">
         <string>P2 multiplier</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
       <property name="text">
         <string>Min Disparity</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="minDisparitySpinBox">
       <property name="toolTip">
         <string>Min Disparity</string>
       </property>
       <property name="minimum">
        <number>-9999</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
       <property name="text">
         <string>Uniqueness Ratio</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="uniquenessRatioSpinBox">
       <property name="toolTip">
         <string>Uniqueness Ratio</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
       <property name="text">
         <string>Speckle Window Size</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="speckleWindowSizeSpinBox">
       <property name="toolTip">
         <string>Speckle Window Size</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>50</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
       <property name="text">
         <string>Speckle Range</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="speckleRangeSpinBox">
       <property name="toolTip">
         <string>Speckle Range</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_8">
       <property name="text">
         <string>Disp 12 Max Diff</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="disp12MaxDiffSpinBox">
       <property name="toolTip">
         <string>Disp 12 Max Diff</string>
       </property>
       <property name="minimum">
        <number>-9999</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
       <property name="text">
         <string>Full DP</string>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QCheckBox" name="fullDPCheckBox">
       <property name="toolTip">
         <string>Full DP</string>
       </property>
       <property name="checked">
           <bool>true</bool>
       </property>
       <property name="tags" stdset="0">
           <string>advanced</string>
       </property>
     </widget>
    </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="3"/>
 <resources/>
 <connections>
  <connection>
   <sender>preFilterCapSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>sADWindowSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>p1MultiplierSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>p2MultiplierSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>minDisparitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>uniquenessRatioSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>speckleWindowSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>speckleRangeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>disp12MaxDiffSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
  <connection>
   <sender>fullDPCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>OpenCVSGMParametersControlWidget</receiver>
   <slot>changeParameters()</slot>
  </connection>
 </connections>
 <slots>
  <slot>changeParameters()</slot>
 </slots>
 <customwidgets>
 </customwidgets>
</ui>
